tutorials/install.md

```diff
@@ -46,6 +46,35 @@ Install Ignition CMake:
 Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
 which version you need.
 
+### Windows
+
+Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+Miniconda suffices.
+
+Create if necessary, and activate a Conda environment:
```

Comment:
Perhaps a note here mentioning the user will need to open a windows command prompt being sure to add `conda` to their PATH?

Reply:
Okay. I duplicated the instructions from the Source Install here in 1d50345. It's a little weird to have duplicate instructions but seems that's clearer.